contracts/actions/bprotocol/BprotocolLiquitySPDeposit.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import "../../interfaces/bprotocol/IBAMM.sol";
+import "../ActionBase.sol";
+import "./helpers/BprotocolLiquitySPHelper.sol";
+import "../../utils/TokenUtils.sol";
+
+
+contract BprotocolLiquitySPDeposit is ActionBase, BprotocolLiquitySPHelper {
+    using TokenUtils for address;
+
+    struct Params {
```

Comment:
Usually we do the amount variable first and the from/to last, switch places to keep variable order consistent between actions

Reply:
Also add natspec for stuff like to explain that during deposit lqty rewards are sent to the proxy

Reply:
will do